index.js

```diff
@@ -512,18 +521,6 @@ let Fixtures = {
             }
         });
     },
-
-    /**
-     * Callback to be performed after a refresh.
-     *
-     * @param {Object} response Chakram refresh response.
-     *
-     * @private
-     */
-    _afterRefresh(response) {
-        this._headers['OAuth-Token'] = response.body.access_token;
-        this._refreshToken = response.body.refresh_token;
-    },
```

Comment:
We don't need this - it's a duplicate of `_storeAuth`.

I think we might have previously decided we might eventually want to have them do different things, but now that Thorn is relatively stable and not likely to undergo massive structural changes I think we can safely get rid of it.